misk/src/main/kotlin/misk/moshi/MoshiExtensions.kt

```diff
@@ -2,4 +2,4 @@ package misk.moshi
 
 import com.squareup.moshi.Moshi
 
-inline fun <reified T> Moshi.adapter() = adapter(T::class.java)
+inline fun <reified T> Moshi.adapter() = this.adapter(T::class.java)!!
```

Comment:
what is happening here?

Reply:
We were getting a warning around unknown nullability of platform types.